README.md

```diff
@@ -45,6 +46,6 @@ I have tried to simplify the code so anyone can read it and start contibuting at
 
 - [ ] Allowing users to choose a reddit thread instead of being randomized.
 - [ ] Allowing users to choose a background that is picked instead of the Minecraft one.
-- [ ] Allowing users to choose between any subreddit.
+- [✅] Allowing users to choose between any subreddit.
```

Comment:
out of the scope of this PR, but I think being able to pull from multiple subreddits at once would also be great! WDYT?